src/Zicht/Bundle/UrlBundle/Command/ImportUrlAliasesCommand.php

```diff
@@ -0,0 +1,201 @@
+        $flush = $aliasingService->setIsBatch(true);
+
+        $handle = fopen($input->getArgument('file'), 'r');
+        if (false === $handle) {
+            throw new \Exception('Can not open input file');
+        }
+
+        $lineNumber = 0;
+
+        if ($input->getOption('skip-header')) {
+            $lineNumber++;
+            $data = fgetcsv($handle, null, $csvDelimiter, $csvEnclosure);
+            if (false === $data) {
+                throw new \Exception(sprintf('Can not read line %s in input file', $lineNumber));
+            }
+        }
+
+        while ($data = fgetcsv($handle, null, $csvDelimiter, $csvEnclosure)) {
+            $lineNumber++;
+
+            if (false === $data) {
+                throw new \Exception(sprintf('Can not read line %s in input file', $lineNumber));
+            }
+
+            if (null === $data || [null] === $data) {
+                // skip empty line
+                continue;
+            }
+
+            if (sizeof($data) < 2) {
+                throw new \Exception('Every line in the file must have at least the publicUrl and internalUrl');
+            }
+
+            $publicUrl = trim($data[0]);
+            $internalUrl = trim($data[1]);
+            $type = $this->parseInputToMapping($this->redirectTypeMapping, $data, 2, $defaultRedirectType);
+            $conflictingPublicUrlStrategy = $this->parseInputToMapping($this->defaultConflictingPublicUrlStrategyMapping, $data, 3, $defaultConflictingPublicUrlStrategy);
+            $conflictingInternalUrlStrategy = $this->parseInputToMapping($this->defaultConflictingInternalUrlStrategyMapping, $data, 3, $defaultConflictingInternalUrlStrategy);
+
+            $realInternalUrl = $aliasingService->hasInternalAlias($internalUrl);
+            if (null !== $realInternalUrl && $internalUrl !== $realInternalUrl) {
+                // the $internalUrl given in the csv is actually known as a public url in our system.
+                // the $publicUrl given in the csv should redirect directly to this known public url.
+                $redirectDescription = sprintf('%s -> %s -> %s', $publicUrl, $internalUrl, $realInternalUrl);
+                $internalUrl = $realInternalUrl;
+            } else {
+                $redirectDescription = sprintf('%s -> %s', $publicUrl, $internalUrl);
+            }
+
+            // perform aliasing
+            $aliasingService->addAlias($publicUrl, $internalUrl, $type, $conflictingPublicUrlStrategy, $conflictingInternalUrlStrategy);
+            $output->writeln(sprintf('%s  %s  %s', $lineNumber, $type, $redirectDescription));
+        }
+
+        $flush();
+    }
+
+    /**
+     * Return the parsed value from $data[$index]
+     *
+     * @param array $mapping
+     * @param array $data
+     * @param integer $index
+     * @param integer $default
+     * @return integer
+     */
+    private function parseInputToMapping($mapping, $data, $index, $default)
```

Comment:
The default value is different for every call made to this method, do default value makes no sense here